package.json

```diff
@@ -19,7 +19,10 @@
     "bluebird": "^3.5.1",
     "daemonize2": "^0.4.2",
     "iobroker.admin": ">=2.0.9",
+    "iobroker.vis-jqui-mfd": "1.0.11",
```

Comment:
I know you didn't add this, but why is a vis addon even being installed by default?

Reply:
It is mistake. I think and we must remove it

Reply:
True